src/scripts/doctor.js

```diff
@@ -0,0 +1,38 @@
+const execa = require("execa");
+const ora = require("ora");
+const path = require("path");
+
```

Comment:
TODO:
* auto fix by -f/--fix flag
* files field
* repository field
* license field
* etc
* if there are issues, should be followed by question to fix it